index.jsx

```diff
@@ -3,13 +3,20 @@ import React, {PropTypes} from 'react';
 function radio(name, selectedValue, onChange) {
   return React.createClass({
     render: function() {
+      const optional = {};
+      if(typeof selectedValue !== 'undefined') {
```

Comment:
Please, check as `selectedValue !== undefined`. StackOverflow answers are not the best, unfortunately.

See https://github.com/lodash/lodash/blob/3.0.1-npm-packages/lodash.isundefined/index.js